src/redux/actions/projects/updateProject.js

```diff
@@ -22,7 +25,11 @@ const updateProject = (
   const newProject = mergeObjectWithArrays(currentProject, diff);
 
   try {
-    await dispatch(saveProject(projectUuid, newProject));
+    if (config.currentApiVersion === api.V1) {
+      await dispatch(saveProject(projectUuid, newProject));
+    } else if (config.currentApiVersion === api.V2) {
+      // Dont do any fetch, updating the experiment is enough
+    }
```

Comment:
Do we need this `else if` block? Can't the comment be put before the `if` statement above explaning why it'so only done for V1?

Reply:
ok